.github/workflows/codeql.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Initialize CodeQL
       uses: github/codeql-action/init@v1

```

Comment:
forgot to mention that both the init and the annalyze should be moved to v2 if possible 😋 
https://github.com/github/codeql-action#usage